shared/profile/edit-avatar/container.tsx

```diff
@@ -68,17 +69,32 @@ export default Container.connect(
       image: stateProps.image,
       onBack: dispatchProps.onBack,
       onClose: dispatchProps.onClose,
-      onSave: (filename: string, crop?: RPCTypes.ImageCropRect) =>
-        wizard
-          ? dispatchProps.onSaveWizardAvatar(filename, crop)
-          : stateProps.teamname
-          ? dispatchProps.onSaveTeamAvatar(
-              filename,
-              stateProps.teamname,
-              stateProps.sendChatNotification,
-              crop
-            )
-          : dispatchProps.onSaveUserAvatar(filename, crop),
+      onSave: (
+        filename: string,
+        crop?: RPCTypes.ImageCropRect,
+        scaledWidth?: number,
+        offsetLeft?: number,
+        offsetTop?: number
+      ) => {
+        if (
```

Comment:
I just moved the first conditional in an `if` block since there were a bunch of extra conditions to check, but could use the original format.